packages/auth/src/hooks/useApiRoutes.ts

```diff
@@ -22,7 +22,7 @@ const apiRoutes = {
 
     const url = apiHost
       ? new URL(loginPath, apiHost)
-      : ["login"].join("/") + (searchString ? `?${searchString}` : "");
+      : ["/login"].join("/") + (searchString ? `?${searchString}` : "");
```

Comment:
I know you didn't write this code but it's a bit silly to call join on an array with one item.  :grin:

Reply:
And why wouldn't you use `loginPath` with a slash like L33?